# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ooo/system-mozilla.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

For me it loooks like standard behaviour that the linker does not investigate
indirect library dependencies to resolve symbols - maybe other system compiler/
linker are strangely patched ... So we specify the dependencies the code
exposes to let the linker succeed linking the final object ...

  - Rene Rebe <rene@exactcode.de>

--- ./patches/src680/system-mozilla-xmlsecurity-libs.diff.orig	2005-04-18 14:06:01.000000000 +0200
+++ ./patches/src680/system-mozilla-xmlsecurity-libs.diff	2005-04-19 09:40:04.000000000 +0200
@@ -0,0 +1,13 @@
+--- solenv/inc/libs.mk.vanilla	2005-04-19 09:17:36.000000000 +0200
++++ solenv/inc/libs.mk	2005-04-19 09:19:26.000000000 +0200
+@@ -155,7 +155,9 @@
+ NSS3LIB=-lnss3
+ NSPR4LIB=-lnspr4
+ PLC4LIB=-lplc4
+-NSSCRYPTOLIBS=$(LIBXML2LIB) $(XMLSECLIB) $(XMLSECLIB-NSS) $(NSS3LIB) $(NSPR4LIB) $(PLC4LIB)
++PLDS4LIB=-lplds4
++SOFTOKN3LIB=-lsoftokn3
++NSSCRYPTOLIBS=$(LIBXML2LIB) $(XMLSECLIB) $(XMLSECLIB-NSS) $(NSS3LIB) $(NSPR4LIB) $(PLC4LIB) $(PLDS4LIB) $(SOFTOKN3LIB)
+ AUTOMATIONLIB=-lsts$(OFFICEUPD)$(DLLPOSTFIX)
+ SVLLIB=-lsvl$(OFFICEUPD)$(DLLPOSTFIX)
+ SVMEMLIB=
